src/pathPlanning/GCodePath.cpp

```diff
@@ -6,6 +6,20 @@
 namespace cura
 {
 
+GCodePath::GCodePath(const GCodePathConfig* config, SpaceFillType space_fill_type, float flow, bool spiralize) :
+config(config),
+space_fill_type(space_fill_type),
+flow(flow),
+spiralize(spiralize)
+{
+    retract = false;
```

Comment:
Why not initialize every member above, rather than in the body of the constructor?